cluster-autoscaler/utils.go

```diff
@@ -187,6 +187,32 @@ func resetPodScheduledConditionForPod(kubeClient *kube_client.Client, pod *kube_
 	return fmt.Errorf("Expected condition PodScheduled")
 }
 
+// FilterOutSchedulable return list of pods which are unschedulable.
```

Comment:
Can you please expand this comment. With pod condition `PodScheduled` it's not obvious what this function is actually doing.